common/djangoapps/cme_registration/tests/registration_tests.py

```diff
@@ -401,10 +381,8 @@ def test_db_records_created(self):
     @unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Run only in LMS')
     def test_db_records_with_others_created(self):
 
-        self.post_vars['specialty'] = 'Other'
-        self.post_vars['specialty_free'] = 'Patient care'
-        self.post_vars['sub_specialty'] = 'Other'
-        self.post_vars['sub_specialty_free'] = 'Legs and feet'
+        self.post_vars['specialty'] = 'Other/None'
```

Comment:
Similar to my comment on #573 , should this entire test case method go away, since we no longer allow creation with "other" values?